ArduPlane/navigation.cpp

```diff
@@ -136,7 +136,30 @@ void Plane::calc_airspeed_errors()
             target_airspeed_cm = ((int32_t)(aparm.airspeed_max - aparm.airspeed_min) *
                                   get_throttle_input()) + ((int32_t)aparm.airspeed_min * 100);
         }
+#if OFFBOARD_GUIDED == ENABLED
+    } else if (control_mode == &mode_guided && !is_zero(guided_state.target_airspeed_cm)) {
+        // offboard airspeed demanded
+        uint32_t now = AP_HAL::millis();
+        float delta = 1e-3f * (now - guided_state.target_airspeed_time_ms);
+        guided_state.target_airspeed_time_ms = now;
+        float delta_amt = 100 * delta * guided_state.target_airspeed_accel;
+        target_airspeed_cm += delta_amt;
+        if (is_positive(guided_state.target_airspeed_accel)) {
+            target_airspeed_cm = MIN(guided_state.target_airspeed_cm, target_airspeed_cm);
+        } else {
+            target_airspeed_cm = MAX(guided_state.target_airspeed_cm, target_airspeed_cm);
+        }
 
+        // precautionary clamp/s to limits, with gcs warning
+        if (  target_airspeed_cm < ( aparm.airspeed_min * 100) ) { 
+            gcs().send_text(MAV_SEVERITY_INFO,"target_airspeed_cm clamped to MIN airspeed");
```

Comment:
this looks like it would flood the link with these STATUSTEXT msgs if it ever happens. Maybe just remove?

Reply:
seems fair, willdo.